src/components/Patients.js

```diff
@@ -1,18 +1,24 @@
-import React from 'react';
+import React, { Component, PropTypes } from 'react';
 
 import PageHeader from '../components/Header/PageHeader';
 import PatientList from '../components/PatientList/PatientList';
 
-const Patients = () => {
-  return (
-    <div className="patients container">
-      <PageHeader title="Patients" />
 
-      <PatientList />
-    </div>
-  );
-};
+class Patients extends Component {
+  render() {
+    return (
+      <div className="patients container">
+        <PageHeader title="Patients" />
+        <PatientList />
+      </div>
+    );
+  }
+}
 
 Patients.displayName = 'Patients';
 
+Patients.propTypes = {
+  location: PropTypes.object
```

Comment:
We should really think about having a `patient.js` prop-type and perhaps a middleware. Having an `object` prop-type isn't super informative. Thoughts?